src/routes/home/HomeView.vue

```
@@ -208,18 +208,27 @@ export default {
           this.handleSubscribeError(Number(this.$route.query.gameId), this.t(this.$route.query.error));
           this.$router.replace('/');
         }
-      }
-    }
+        if ([ 'discord' ].includes(this.$route.query?.oauthsignup)) {
+          this.oAuthSignup = true;
+          this.$router.replace('/');
+        }
+      },
+    },
   },
   async created() {
-    await this.gameListStore.requestGameList();
-    this.loadingData = false;
-    this.tab = this.$route.path;
+    try {
+      await this.gameListStore.requestGameList();
+    } catch (err) {
+      console.log(err);
```

We have to swallow the error so the oauthsignup can call requestGameList